### PR TITLE
Bug 1853711: Invert http/2 kill switch logic

### PR DESCRIPTION
**Pull request by frobware**
Annotation has been renamed to:

    ingress.operator.openshift.io/default-enable-http2

and by default is "false".

### CODE REVIEW

**Comment by frobware** [comment]
I was unable to test this locally as my cluster timedout. Running in CI whilst I stand up another.

File: pkg/operator/controller/ingress/deployment.go [hunk 1]
```diff
@@ -85,32 +85,32 @@ func (r *reconciler) ensureRouterDeleted(ci *operatorv1.IngressController) error
...
+	if val, ok := m[RouterDefaultEnableHTTP2Annotation]; ok {
 		v, _ := strconv.ParseBool(val)
 		return true, v
 	}
 	return false, false
 }
 
-// HTTP2IsDisabled returns true if the ingress controller disables
+// HTTP2IsEnabled returns true if the ingress controller disables
 // http/2 via the RouterDisableHTTP2Annotation, or if the ingress
```
Comment (wking) [inline pkg/operator/controller/ingress/deployment.go:100]: some stale references to the "disable" default in here.

**Comment by sdodson** [comment]
/hold
Clear this when you're happy with your testing efforts.

**Pull request merged**